bzzeth/peer.go

```diff
@@ -151,10 +162,11 @@ func (r *requests) get(id uint32) (*request, bool) {
 
 // getBlockHeaders sends a GetBlockHeaders message to the remote peer requesting headers by their _hashes_
 // and delivers the actual block header responses to the deliveries channel
-func (p *Peer) getBlockHeaders(ctx context.Context, hashes [][]byte, deliveries chan []byte) (*request, error) {
-	req, id := p.requests.create(hashes, deliveries)
+func (p *Peer) getBlockHeaders(ctx context.Context, hashes []chunk.Address, deliveries chan []byte,
```

Comment:
also just passing one delivery channel should be suffficient

Reply:
yeah.. everything is one channel now